src/assets/GlobAsset.js

```diff
@@ -12,8 +12,15 @@ class GlobAsset extends Asset {
   }
 
   async load() {
-    let files = await globPromise(this.name, {strict: true, nodir: true});
-    let re = micromatch.makeRe(this.name, {capture: true});
+    let regularExpressionSafeName = this.name;
+    if (process.platform === 'win32')
+      regularExpressionSafeName = regularExpressionSafeName.replace(/\\/g, '/');
+
```

Comment:
`\` is an escape in regex; converting to unix style paths here. The `glob` module does the transform internally but `micromatch` doesn't. Doing it here ensures consistency. This was more than just the tests, globs didn't work on Windows.